utils/llm_config.py

```
@@ -0,0 +1,105 @@
+        "zai": "zai",
+        "glm": "zai",
+        "claude": "anthropic",
+        "anthropic": "anthropic",
+        "openai": "openai",
+    }
+    provider = aliases.get(provider, provider)
+    if provider not in SUPPORTED_LLM_PROVIDERS:
+        raise ValueError(
+            f"Unsupported LLM provider {provider!r}. Expected one of: "
+            f"{', '.join(sorted(SUPPORTED_LLM_PROVIDERS))}"
+        )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)